tests/src/chanjo2/endpoints/test_intervals.py

```diff
@@ -214,6 +206,28 @@ def test_transcripts_multiple_filters(
     assert result["detail"] == MULTIPLE_PARAMS_NOT_SUPPORTED_MSG
 
 
+@pytest.mark.parametrize("build", Builds.get_enum_values())
+def test_transcripts_no_filters(
+    build: str,
+    demo_client: TestClient,
+    endpoints: Type,
+    genomic_ids_per_build: Dict[str, List],
+):
+    """Test the endpoint that returns database transcripts without any filter."""
+
+    # GIVEN a populated demo database
+    # WHEN sending a request to the "transcripts" endpoint
+    data = {
+        "build": build,
+    }
+    response: Response = demo_client.post(endpoints.TRANSCRIPTS, json=data)
+
+    # THEN transcript documents should be returned
```

Comment:
```suggestion
    # THEN transcript objects should be returned
```
Not documents right? If correct change for the other tests

Reply:
Yes, objects. Will do, thanks!